### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by Morganmrose** [ConnorMacDonell/COMP-330-Project1#5]
As your project grows, it may be better to add more classes instead of having everything in the main function. Your Readme contains good background information regarding the twitter implementations and how your functions are to handle them. However, it may be more useful to you to comment within your code what your functions are doing so you can keep better track of what these functions are doing. It will also help others see what your functions are doing. 

May also be good to add more complexity to the tests, such as the counters. For instance, something I did as well (and would like to change), is just count the number of times a "#" appeared, instead of checking the characters/string/numbers after it. If Twitter only topics what is followed after the hashtag, would it count "##" as two or one topics, or none, since it only contains hashtags and no real content? Not very familiar with Twitter, but it definitely seems like this could be a good implementation to think about.

**Comment by cartfisk** [comment]
I agree with your comment about the readme, I think there is a lot of information in there that would be better suited as comments adjacent to actual code so you can see what pieces are actually doing what you write about in the readme. The readme itself should focus more on things like project scope, setup instructions, dependencies, testing instructions, external references etc.
